crates/task-impls/src/network.rs

```diff
@@ -174,12 +177,24 @@ impl<TYPES: NodeType> NetworkMessageTaskState<TYPES> {
                         warn!("Request and Response messages should not be received in the NetworkMessage task");
                     }
                 },
+
+                MessageKind::External(data) => {
+                    // Send the external message to the external event stream so it can be processed
+                    broadcast_event(
+                        Event {
+                            view_number: TYPES::Time::new(1),
```

Comment:
How come this is view 1?

Reply:
It's arbitrary, it doesn't necessarily need a view number so I set it to one; I followed what `ResponseMessage::Denied` was doing. It might make sense to use the current view but I don't want it to cause confusion